src_files.yml

```diff
@@ -1,5 +1,9 @@
 axi_node:
+  vlog_opts: [
+    -L axi_lib
```

Comment:
I do not understand how this repository would result in an `axi_lib` that needs to be searched so that the remaining code in this repository can be optimized. Can you explain?

I can not post a `vlog` script here, but first `vlog`ing `axi` (which is a prerequisite for this repository, see dependency explanation above) and then (this repository into a single `work` directory will work just fine.